app/src/main/java/edu/uc/forbesne/shoppingsidekick/dao/IProductDAO.kt

```diff
@@ -5,7 +5,8 @@ import retrofit2.Call
 import retrofit2.http.GET
 import retrofit2.http.Query
 
-//I used the professor's code from his github - https://github.com/discospiff/MyPlantDiaryQ
+// Spaces after all comments
+// I used the professor's code from his github - https://github.com/discospiff/MyPlantDiaryQ
```

Comment:
Putting conversation/discussion  in code comments is bad practice, as that clutters up the source code and does not belong in source code files.  I mentioned that in the code review instructions.    Discussion and comments are very good, but have that in the appropriate place, which is in the "Files Changed" tab in the pull request in GitHub (which is where I am writing this and you are reading this).  That way, you can have a conversation with replies, right where the code is happening, and without cluttering up the source code itself.